manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "id": "index-coop",
+  "name": "Index Coop",
+  "url": "https://eth-dapp-browser-ne49wqncp-ledgerhq.vercel.app",
+  "params": {
+    "dappUrl": "https://index-app-git-ledger-integration-index-coop.vercel.app/?embedded=true",
+    "nanoApp": "Ethereum",
+    "dappName": "Index Coop",
+    "networks": [
+      {
+        "currency": "ethereum",
+        "chainID": 1,
+        "nodeURL": "wss://eth-mainnet.g.alchemy.com/v2/EElfXQcZk-acQCjrLv4ERIBBdCBxCY2_"
+      },
+      {
+        "currency": "polygon",
+        "chainID": 137,
+        "nodeURL": "wss://polygon-mainnet.g.alchemy.com/v2/svOOPKaFDBQeRA9RaiPoMGCXamTiV0iK"
+      }
+    ]
+  },
+  "homepageUrl": "https://indexcoop.com",
+  "platform": "all",
+  "apiVersion": "0.0.1",
+  "manifestVersion": "1",
+  "branch": "debug",
+  "categories": ["tools"],
+  "currencies": "*",
+  "content": {
+    "shortDescription": {
+      "en": "Decentralized structured products that make crypto simple, accessible, and secure"
+    },
+    "description": {
+      "en": "DeFi doesn’t have to be complicated. Unlock powerful sector, leverage and yield-earning strategies with our simple index tokens."
+    }
+  },
+  "permissions": [
+    {
+      "method": "*"
+    }
+  ],
+  "domains": ["https://*"]
+}
```

Comment:
I guess this manifest needs to be different from the regular one? Otherwise, it would have to be somewhat "merged" with the one under `/public`

Reply:
This manifest is purely for ledger live purposes, I don't think it is related to the one in `public` (although I don't know what the lattter one is used fore)

Reply:
I moved it to `ledger_live/manifest.json` to make this distinction clearer.